server/parsec/components/postgresql/migrations/0017_pki_enrollment_accept_payload_signature_algorithm.sql

```diff
@@ -0,0 +1,12 @@
+-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
```

Comment:
I think we should merge this migration with the previous one (i.e. `0016_pki_enrollment_submit_payload_signature_algorithm.sql`) since:
- migration 0016 is not yet in production
- both migrations basically deal with the same thing (i.e. adding `PKI_SIGNATURE_ALGORITHM` type and using it)